README.adoc

```diff
@@ -103,10 +102,10 @@ You will then manage your deployed microservices using the `kubectl` command lin
 interface for {kube}. The `kubectl` CLI is your primary tool for communicating with and managing your
 {kube} cluster.
 
-The two microservices you will deploy are called `name` and `ping`. The `name` microservice simply
-displays a brief greeting and the name of the
-container that it runs in, making it easy to distinguish it from its other replicas. The `ping` microservice
-simply pings the {kube} Service that encapsulates the pods running the `name` microservice. This demonstrates
+The two microservices you will deploy are called `system` and `inventory`. The `system` microservice simply
+returns the JVM systems properties of the container it is running in and it returns the Pod's name in the HTTP header
```

Comment:
- [ ] `the JVM systems properties of the container it is running in` to `the JVM system properties of the running container`?